product-pricing-service/src/main/java/com/hedvig/contract/shared/aggregates/TerminateContractResponse.kt

```diff
@@ -0,0 +1,7 @@
+package com.hedvig.contract.shared.aggregates
+
+enum class TerminateContractResponse {
+    TERMINATED,
+    ALREADY_TERMINATED,
+    COULD_NOT_BE_TERMINATED
```

Comment:
You know all the outcomes given the exceptions so might as well have them clear maybe
```suggestion
    NOT_ACTIVE
```